common_spree_dependencies.rb

```diff
@@ -40,7 +40,6 @@
   gem 'mutant-rspec', '~> 0.8'
   gem 'shoulda-matchers', '~> 3.1'
   gem 'shoulda-callback-matchers', '~> 1.1'
-  gem 'test_after_commit', '~> 1.1'
```

Comment:
@damianlegawiec `test_after_commit` is no longer needed on rails 5.0+
https://github.com/grosser/test_after_commit/blob/master/Readme.md